doc/man1/flux-jobs.rst

```diff
@@ -51,6 +51,13 @@ OPTIONS
    List jobs in a specific queue or queues. Multiple queues may be separated
    by a comma or by using the :option:`-q, --queue` option multiple times.
 
+.. options:: -i, --include=HOSTS|RANKS
+
+   List only jobs where the assigned resources intersect with the supplied
+   argument, which may be specified either as an idset of broker ranks or
+   list of hosts in hostlist form. It is not an error to specify ranks or
+   hosts which do not exist.
```

Comment:
Might want to use "RFC 22 idset" and "RFC 29 hostlist" and add the references to a FLUX RFC section e.g.
```
FLUX RFC
========

| :doc:`rfc:spec_22`
| :doc:`rfc:spec_29`
```

Reply:
Good suggestion! I've done that and also fixed a typo and force-pushed the result.